proposals/53:kread-start/core-eval-support.js

```diff
@@ -0,0 +1,152 @@
+// @ts-check
+import {
+  Far,
+  makeMarshal,
+  makeTranslationTable,
+} from '../../upgrade-test-scripts/lib/unmarshal.js';
+import { Fail, NonNullish } from '../../upgrade-test-scripts/lib/assert.js';
+
+// TODO: factor out ambient authority from these
```

Comment:
I don't think this is necessary for tests. The arguments I've heard for minimizing ambient authority don't apply:
- testability (this is the test itself)
- security (this is a test environment)

The levels of indirection with `io` in this repo are just cognitive overhead imo

Reply:
Yes, we haven't made much progress in convincing each other.

Reply:
I see it the other way around: it's not necessary to sprinkle ambient authority all over test code. That would be more cognitive overhead, for me.